lib/analytics.js

```diff
@@ -463,7 +464,15 @@ Analytics.prototype.page = function(category, name, properties, options, fn) {
     name: name
   });
 
-  this._invoke('page', new Page(msg));
+  var pageFacade = new Page(msg);
+  if (this._pageCalledYet) {
+    pageFacade.isFirstPageCall = false;
```

Comment:
Does this change the output of `pageFacade.toJson()`? i.e. will `isFirstPageCall` field appear in the JSON output? If so, we need to be careful here for a bunch of integrations - the segment one in particular will start sending a new top level field to our API.